packages/unity-bootstrap-theme/stories/docs/GetStarted/get-started.stories.mdx

```diff
@@ -0,0 +1,91 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Get Started/Get Started" />
+
+# ASU Unity Bootstrap Theme (UDS)
+## Quickstart Guide
+
+Using the Unity Bootstrap Theme for building ASU Web Standards 2.0 compliant web sites and apps.
+
+## ❯ What is Unity?:
+
+A Bootstrap 5-based web ui kit for website developers building new websites from scratch or for inclusion as a standardized theming foundation for ASU application frameworks, such as Drupal, WordPress, or other application frameworks.
+
+### Features:
+
+- **Bootstrap 5:** Unity is built on top of Bootstrap 5, so you get all the Bootstrap goodness, plus some extra ASU-specific components and styles. **Bootstrap files,including CSS and Javascript, are bundled with Unity** so you don't need to include them separately.
+- **Clean, easy to use styles and scripts:** thanks to the helper utilities and variables provided by the Bootstrap 5 team.
+- **Font Awesome is no longer included with unity!** If using in a project outside of webspark, you will need to include Font Awesome yourself. See [here](https://fontawesome.com/start) for instructions on how to do that.
+
+## ❯ Including Unity in your project:
+
+We bundle the necessary CSS  and Javascript in the `dist/` folder. There are more options so you don't have to include CSS you don't use.
+
+* `dist/css/unity-bootstrap-theme.bundle.css` - base theme with header and footer CSS
+* `dist/css/unity-bootstrap-theme.css` - base theme without header and footer CSS
+* `dist/css/unity-bootstrap-header.css` (optional) - header CSS only - don't use if using the Unity `component-header` React component
+* `dist/css/unity-bootstrap-footer.css` (optional) - footer CSS only - don't use if using the Unity `component-footer` React component
+
```

Comment:
Let's add a section here:
```
## > How to  use the Unity Storybook reference site

The Storybook reference site for the Unity Bootstrap Theme exists to provide a reference library of components and the markup to use to implement your own instances of the components in your site once you've included the CSS and Javascript in your site or application.
```
Or something similar. Always good to connect all the dots in a Getting Stared document.